src/utils/Config.tsx

```diff
@@ -38,40 +40,50 @@ export const geoUrl =
 /**
  * Default values
  */
-export const defaultCoordinates: Point = [0, 0];
-export const defaultRows: Types.row[] = [];
-export const defaultIndex: number = -1;
-export const defaultVisibility = undefined;
-export const defaultCombinedRows: Types.combinedRow[] = [];
+const defaultCoordinates: Point = [0, 0];
+const defaultRows: Types.Row[] = [];
+const defaultIndex: number = -1;
+const defaultVisibility = undefined;
+export const defaultCombinedRows: Types.CombinedRow[] = [];
 export const defaultTooltipContent: string = "";
-export const defaultCombinedRow: Types.combinedRow = {
+export const defaultCombinedRow: Types.CombinedRow = {
   averageCoordinates: defaultCoordinates,
   rows: defaultRows,
   index: defaultIndex,
   isMarkerVisible: defaultVisibility,
 };
-export const defaultMousePosition: Types.position = {
+const defaultMousePosition: Types.Position = {
   coordinates: defaultCoordinates,
   zoom: minZoom,
 };
 export const defaultSearchBarContent: string = "";
-export const defaultToggle: boolean = false;
-export const defaultState: Types.state = {
+const defaultToggle: boolean = false;
+const defaultMode: boolean = StateUpdaters.checkInFullMode();
+export const defaultRow: Types.Row = {
```

Comment:
Related to "Clicked table row triggers highlight/details/scroll only for that row and not its siblings"